README.md

```diff
@@ -247,9 +258,59 @@ model.set_params(arg_params=arg_params, aux_params=aux_params)
 # Train model
 model.fit(train_data,
           optimizer=opt,
-          opitmizer_params=optimizer_params,
           num_epoch=num_epoch)
+```
+
+###### Gluon Trainer API
+```python
+from mxnet import autograd, gluon
+import mxnet as mx
+import horovod.mxnet as hvd
+
+# Initialize Horovod
+hvd.init()
+
+# Pin GPU to be used to process local rank
+context = mx.gpu(hvd.local_rank())
+num_workers = hvd.size()
 
+# Build model
+model = ...
+model.hybridize()
+
+# Define hyper parameters
+optimizer_params = ...
+
+# Add Horovod Distributed Optimizer
+opt = mx.optimizer.create('sgd', **optimizer_params)
+opt = hvd.DistributedOptimizer(opt)
+
+# Initialize parameters
+model.initialize(initializer, ctx=context)
+
+# Fetch and broadcast parameters
+params = model.collect_params()
+if params is not None:
+    hvd.broadcast_parameters(params, root_rank=0)
+
+# Create trainer and loss function
+trainer = gluon.Trainer(params, opt, kvstore=None)
```

Comment:
Is kvstore=None required? Can we leave it as default?

Reply:
I think it is not required but better to have. We already wrapped the opt with DistributedOptimizer so the kvstore code path will not be executed during training. However, explicitly setting it to None will skip some unnecessary checking internally. And it is also informative that we do not use kvstore with Horovod.